project_task_stock_product_set/readme/CONFIGURE.md

```diff
@@ -0,0 +1,5 @@
+It is important to clarify some concepts at the level of project stages:
```

Comment:
This change from RST to MD must be in the `pre-commit autofixes` commit, not in the migration one.